### PR TITLE
Anomali solved

### PR DESCRIPTION
**Pull request by roysagi**
## Status
Ready

## Description
Dropped the '.' in the context created - this seems to be the cause of the index out of range error in this command: !threatstream-email-reputation email=mailonline_16@filposcv.com

## Does it break backward compatibility?
   - Yes
       - Further details: @yaakovi suggested it might need to see in circle
   - No

### CODE REVIEW

**Comment by roysagi** [comment]
@yaakovi - can you say why do you think it might break backwards compatibility?

**Comment by Itay4** [comment]
because users who use populate context path will no longer be able to do so
need to populate to both context paths and change to the correct one (without the trailing dot) in the integration settings

**Comment by roysagi** [comment]
this context path causes error of `index out of range` on server side when running the command: `!threatstream-email-reputation email=mailonline_16@filposcv.com'

After the change context is created correctly
this is causing the command to return both valid output and error entry (of runtime error)   - causing any playbook to stop.

As you can see - context is not created *before* this change was made:
Screen Shot 2019-08-01 at 18 25 34

Screen Shot 2019-08-01 at 18 22 32

Thus this should not cause BC errors.

File: Releases/LatestRelease/integration-AnomaliThreatStream.md [hunk 1]
```diff
@@ -0,0 +1 @@
+In line 251 - dropped the '.' in the end of the word 'Reputation'
```
Comment (yaakovi) [inline Releases/LatestRelease/integration-AnomaliThreatStream.md]: @kirbles19 please fix this release notes. they are too developer oriented.

**Comment by yaakovi** [comment]
since the context is not created and the original functionality causes errors, this shouldn't be considered as a BC breaking.

**Comment by kirbles19** [comment]
@roysagi @yaakovi RN updated, all done.

**Comment by roysagi** [comment]
It seems Anomali currently has an unrelated issue concerning credentials (https://github.com/demisto/etc/issues/18561) thus, Anomali's test has been skipped.
I've consulted with @yaakovi and because this issue was solved and tested before the credential problem - I'll still merge this fix.

**Comment by yaakovi** [comment]
@roysagi pull from master and update the RN file to the new standard (*CHANGELOG.md*)

**Pull request merged**